lib/index.js

```diff
@@ -2,6 +2,8 @@ const parseJSON = require('json-parse-even-better-errors')
 const { diff } = require('just-diff')
 const { diffApply } = require('just-diff-apply')
 
+const globalObjectProperties = Object.getOwnPropertyNames(Object.prototype)
```

Comment:
I have no idea how hot this path is so this is admittedly a premature optimization, but it seems like converting this to a map/object once and then using `.has` might be a good idea.

Reply:
This is not a very hot path, and the diff path is normally not deep.

Reply:
After checking, we only call this 1-2 times for an entire process execution, so this is fine to leave as is.